src/github/createPRViewProvider.ts

```diff
@@ -112,27 +112,40 @@ export class CreatePullRequestViewProvider extends WebviewViewBase implements vs
 		super.show();
 	}
 
+	private async getTotalCommits() {
+		const origin = await this._folderRepositoryManager.getOrigin(this.compareBranch);
+
+		if (this.compareBranch?.upstream) {
+			const headRepo = this._folderRepositoryManager.findRepo(byRemoteName(this.compareBranch.upstream.remote));
+
+			if (headRepo) {
+				const headBranch = `${headRepo.remote.owner}:${this.compareBranch.name ?? ''}`;
+				const baseBranch = `${this._pullRequestDefaults.owner}:${this._pullRequestDefaults.base}`;
+				const { total_commits } = await origin.compareCommits(baseBranch, headBranch);
+
+				return total_commits;
+			}
+		}
+
+		return 0;
```

Comment:
I wasn't too sure how to handle the case where `this.compareBranch == null`. Advice appreciated!

Reply:
When the `upstream` is undefined we can use the git API to get the number of changes. I have added that.